src/main/java/ch/naviqore/raptor/RaptorAlgorithm.java

```diff
@@ -0,0 +1,49 @@
+package ch.naviqore.raptor;
+
+import ch.naviqore.raptor.impl.RaptorBuilder;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+public interface RaptorAlgorithm {
+
+    static RaptorBuilder builder(int sameStopTransferTime) {
```

Comment:
This doesn't really work looking forward. I think the raptor builder is too much implementation specific that we can keep it in the interface. Anyway at the moment the Raptor Builder is inside the Impl package.

Reply:
Agree, I think we will have to implement a factory for the Raptor.